odoo/tools/osutil.py

```diff
@@ -54,34 +54,6 @@ def clean_filename(name, replacement=''):
         return "Untitled"
     return re.sub(r'[^\w_.()\[\] -]+', replacement, name).lstrip('.-') or "Untitled"
 
-def listdir(dir, recursive=False):
```

Comment:
It seems still used by  `get_module_filetree` (used by `download_iot_handlers`) :/

Reply:
Yep yer right, I'd only deprecated the non-recursive version so I should only remove that bit.

And probably deprecate `get_module_filetree` (and recursive `listdir`), and rewrite `download_iot_handlers` to stop using that. Which means I need to understand what this mess does.

:crying_cat_face: